optaweb-tsp-planner/src/test/java/org/optaweb/tsp/optawebtspplanner/OptawebTspPlannerApplicationTests.java

```diff
@@ -43,6 +44,7 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @AutoConfigureMockMvc
+@ActiveProfiles("test")
```

Comment:
In general, String dependency does not provide any type safety. Would it be possible to define a constant and use it in all places we refer to the "test" profile?